docs/pages/server-access/guides/bpf-session-recording.mdx

```diff
@@ -28,6 +28,27 @@ ingest and perform monitoring and alerting on.
 
 Teleport Enhanced Session Recording mitigates all three concerns by providing advanced security and greater logging capabilities, and better correlates a user with their activities.
 
+<Notice type="warning">
```

Comment:
Just checking to see if you've compared using a Notice to an Admonition here. This is quite a long paragraph, so an Admonition might look more elegant

Reply:
Thanks for the tip! Just swapped it over.